### PR TITLE
Stitches KAWS

### PR DESCRIPTION
**Pull request by l2succes**
Depends on: https://github.com/artsy/kaws/pull/4

Adds support for the following query
```graphql
{
  marketingCollection(slug: "kaws-companions") {
    id
    title
    query {
      artist_ids
      artist_id
      gene_id
    }
    artworks {
      __id
      hits {
    	id
        title
      }
    }
  }
}
```
screen shot 2018-10-11 at 3 43 40 pm

### CODE REVIEW

File: src/lib/stitching/mergeSchemas.ts [hunk 1]
```diff
@@ -48,6 +51,20 @@ export const incrementalMergeSchemas = (testConfig?: any) => {
     }
   }
 
+  if (ENABLE_KAWS_STITCHING) {
```
Comment (orta) [inline src/lib/stitching/mergeSchemas.ts]: TBH, you can probably skip this env var and always stitch 👍

Comment (l2succes) [reply src/lib/stitching/mergeSchemas.ts]: True, I'm just gonna leave it for now as I don't have KAWS setup on production yet

Comment (orta) [reply src/lib/stitching/mergeSchemas.ts]: Cool - your errors are beast the `.env.testing` doesn't have the URLs set

Comment (l2succes) [reply src/lib/stitching/mergeSchemas.ts]: Yeah, I noticed, just set the variable via hokusai but I may have to set a default 🤔

Comment (l2succes) [reply src/lib/stitching/mergeSchemas.ts]: Nevermind, I see what you're saying... just updated `.env.test`

**Pull request merged**